README.md

```diff
@@ -820,7 +820,11 @@ Notes:
     - works only for *multi-gpu*.
     - currently only includes the version of *multipack* optimized for linear attention implementations like *flash-attn*.
  * Notes on Fast MoE
-    - `--fast_moe` is an integer value that configures the amount of expert parallel sharding (ep_degree).
+    - `--fast_moe` takes either an integer or boolean value.
```

Comment:
Not really your change but could you expand the line on `803` to put more details on what FAST MOE does...I believe that's where we first talk about it and then jump into expert parallel degrees here also the link is to code not a document in fms-acceleration so that doesn't help either.

Reply:
>  trains MoE models in parallel with scatter MoE kernels, increasing throughput and decreasing memory usage.

Seems legible to me from a user guide point.